lametro/models.py

```diff
@@ -838,7 +838,7 @@ def is_upcoming(self):
     def is_ongoing(self):
         return self in type(self)._streaming_meeting()
 
-    @property
+    @cached_property
```

Comment:
How does this work?

Reply:
`cached_property` caches a property value for just for the life of a model instance.

Reply:
Oh, sick!